app/javascript/Components/student_table.jsx

```diff
@@ -225,14 +227,17 @@ class RawStudentTable extends React.Component {
               filterable: false,
             },
           ]}
+          noDataText={I18n.t("students.empty_table")}
```

Comment:
This line should be moved back to Line 240 (look at where it is in the original code), as there isn't actually a change.